.github/workflows/build.yml

```diff
@@ -44,7 +48,7 @@ jobs:
           path: cmd/ipfs/ipfs
   interop:
     needs: [interop-prep]
-    runs-on: ubuntu-latest
+    runs-on: ${{ fromJSON(github.repository == 'ipfs/kubo' && '["self-hosted", "linux", "x64", "2xlarge"]' || '"ubuntu-latest"') }}
```

Comment:
Just curious what ec2 instance type is "2xlarge" and "4xlarge" ?

Reply:
These are from the C5 family in AWS - https://github.com/pl-strflt/tf-aws-gh-runner#runner-types